package.json

```diff
@@ -8,7 +8,14 @@
     "db:migrate": "knex migrate:latest",
     "db:seed": "knex seed:run",
     "start": "node index.js",
-    "dev": "nodemon index.js"
+    "dev": "nodemon index.js",
+    "dev2": "node --watch index.js",
+    "migrate": "npx knex migrate:latest",
+    "rollback": "npx knex migrate:rollback",
+    "migrate:rollback": "knex migrate:rollback",
+    "drop": "npx knex migrate:down",
+    "migrate:down": "knex migrate:down",
```

Comment:
is there any difference in functionality between lines 14&15, lines 16&17, lines 8&13 and lines 9&18?

Reply:
You're right!
Deleted line 15, 17, 8, 9